src/modm/ui/color/rgbhtml.hpp

```diff
@@ -0,0 +1,190 @@
+	static constexpr RgbHtml BurlyWood()		{ return RgbHtml(222, 184, 135); };
+	static constexpr RgbHtml Tan()				{ return RgbHtml(210, 180, 140); };
+	static constexpr RgbHtml RosyBrown()		{ return RgbHtml(188, 143, 143); };
+	static constexpr RgbHtml SandyBrown()		{ return RgbHtml(244, 164, 96); };
+	static constexpr RgbHtml Goldenrod()		{ return RgbHtml(218, 165, 32); };
+	static constexpr RgbHtml DarkGoldenrod()	{ return RgbHtml(184, 134, 11); };
+	static constexpr RgbHtml Peru()				{ return RgbHtml(205, 133, 63); };
+	static constexpr RgbHtml Chocolate()		{ return RgbHtml(210, 105, 30); };
+	static constexpr RgbHtml SaddleBrown()		{ return RgbHtml(139, 69, 19); };
+	static constexpr RgbHtml Sienna()			{ return RgbHtml(160, 82, 45); };
+	static constexpr RgbHtml Brown()			{ return RgbHtml(165, 42, 42); };
+	static constexpr RgbHtml Maroon()			{ return RgbHtml(128, 0, 0); };
+// 	// White HTML Color Names
+	static constexpr RgbHtml White()			{ return RgbHtml(255, 255, 255); };
+	static constexpr RgbHtml Snow()				{ return RgbHtml(255, 250, 250); };
+	static constexpr RgbHtml HoneyDew()			{ return RgbHtml(240, 255, 240); };
+	static constexpr RgbHtml MintCream()		{ return RgbHtml(245, 255, 250); };
+	static constexpr RgbHtml Azure()			{ return RgbHtml(240, 255, 255); };
+	static constexpr RgbHtml AliceBlue()		{ return RgbHtml(240, 248, 255); };
+	static constexpr RgbHtml GhostWhite()		{ return RgbHtml(248, 248, 255); };
+	static constexpr RgbHtml WhiteSmoke()		{ return RgbHtml(245, 245, 245); };
+	static constexpr RgbHtml SeaShell()			{ return RgbHtml(255, 245, 238); };
+	static constexpr RgbHtml Beige()			{ return RgbHtml(245, 245, 220); };
+	static constexpr RgbHtml OldLace()			{ return RgbHtml(253, 245, 230); };
+	static constexpr RgbHtml FloralWhite()		{ return RgbHtml(255, 250, 230); };
+	static constexpr RgbHtml Ivory()			{ return RgbHtml(255, 255, 240); };
+	static constexpr RgbHtml AntiqueWhite()		{ return RgbHtml(250, 235, 215); };
+	static constexpr RgbHtml Linen()			{ return RgbHtml(250, 240, 230); };
+	static constexpr RgbHtml LavenderBlush()	{ return RgbHtml(255, 240, 245); };
+	static constexpr RgbHtml MistyRose()		{ return RgbHtml(255, 228, 225); };
+
+	// Gray HTML Color Names
+	static constexpr RgbHtml Gainsboro()		{ return RgbHtml(220, 220, 220); };
+	static constexpr RgbHtml LightGray()		{ return RgbHtml(211, 211, 211); };
+	static constexpr RgbHtml Silver()			{ return RgbHtml(192, 192, 192); };
+	static constexpr RgbHtml DarkGray()			{ return RgbHtml(169, 169, 169); };
+	static constexpr RgbHtml Gray()				{ return RgbHtml(128, 128, 128); };
+	static constexpr RgbHtml DimGray()			{ return RgbHtml(105, 105, 105); };
+	static constexpr RgbHtml LightSlateGray()	{ return RgbHtml(119, 136, 153); };
+	static constexpr RgbHtml SlateGray()		{ return RgbHtml(112, 128, 144); };
+	static constexpr RgbHtml DarkSlateGray()	{ return RgbHtml(47, 79, 79); };
+	static constexpr RgbHtml Black()			{ return RgbHtml(0, 0, 0); };
```

Comment:
I would prefer a simple static constexpr variable, instead of function. There's no benefit to the function and it's weird to have them like this.

I also don't see a benefit of scoping them in a class. Perhaps something as simple as a `modm::color::html` namespace with these constexpr Rgb8 colors would suffice?